feg/gateway/plmn_filter/plmn_filter_test.go

```diff
@@ -0,0 +1,51 @@
+package plmn_filter
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	plmns_5       = []string{"00102", "00103"}
+	plmns_6       = []string{"008043", "009899"}
+	plmns_5_and_6 = []string{"00102", "001027"}
+)
+
+func TestPlmn5(t *testing.T) {
+	plmnIdVals := GetPlmnVals(plmns_5)
+
+	assert.True(t, CheckImsiOnPlmnIdListIfAny("001020000000055", plmnIdVals),
+		"IMSI 001020000000055 should be valid PLMN, PLMN ID Map", plmnIdVals)
+	assert.True(t, CheckImsiOnPlmnIdListIfAny("001030000000055", plmnIdVals),
+		"IMSI 001030000000055 should be valid PLMN, PLMN ID Map", plmnIdVals)
+	assert.False(t, CheckImsiOnPlmnIdListIfAny("001010000000055", plmnIdVals),
+		"IMSI 001010000000055 should NOT be valid PLMN, PLMN ID Map", plmnIdVals)
+
+}
+
+func TestPlmn6(t *testing.T) {
+	plmnIdVals := GetPlmnVals(plmns_6)
+
+	assert.True(t, CheckImsiOnPlmnIdListIfAny("008043000000055", plmnIdVals),
+		"IMSI 008043000000055 should be valid PLMN, PLMN ID Map: %+v", plmnIdVals)
+	assert.True(t, CheckImsiOnPlmnIdListIfAny("009899000000055", plmnIdVals),
+		"IMSI 009899000000055 should be valid PLMN, PLMN ID Map: %+v", plmnIdVals)
+	assert.False(t, CheckImsiOnPlmnIdListIfAny("008040000000055", plmnIdVals),
+		"IMSI 008040000000055 should NOT be valid PLMN, PLMN ID Map: %+v", plmnIdVals)
+
+}
+
+func TestPlmn6and6(t *testing.T) {
+	plmnIdVals := GetPlmnVals(plmns_5_and_6)
+
+	assert.True(t, CheckImsiOnPlmnIdListIfAny("001023000000055", plmnIdVals),
+		"IMSI 001023000000055 should be valid PLMN, PLMN ID Map: %+v", plmnIdVals)
+	assert.True(t, CheckImsiOnPlmnIdListIfAny("001027000000055", plmnIdVals),
+		"IMSI 001027000000055 should be valid PLMN, PLMN ID Map: %+v", plmnIdVals)
+	assert.True(t, CheckImsiOnPlmnIdListIfAny("001020000000055", plmnIdVals),
+		"IMSI 001020000000055 should NOT be valid PLMN, PLMN ID Map: %+v", plmnIdVals)
+	assert.False(t, CheckImsiOnPlmnIdListIfAny("001090000000055", plmnIdVals),
+		"IMSI 001090000000055 should NOT be valid PLMN, PLMN ID Map: %+v", plmnIdVals)
+
```

Comment:
the spacing looks a little weird here, can we get rid of the empty line at the bottom of the tests?